lldb/source/Interpreter/Options.cpp

```diff
@@ -1410,7 +1410,9 @@ llvm::Error lldb_private::CreateOptionParsingError(
     llvm::StringRef long_option, llvm::StringRef additional_context) {
   std::string buffer;
   llvm::raw_string_ostream stream(buffer);
-  stream << "Invalid value ('" << option_arg << "') for -" << short_option;
+  stream << "Invalid value ('" << option_arg << "')";
```

Comment:
"start the first sentence with a lowercase letter"
```suggestion
  stream << "invalid value ('" << option_arg << "')";
```